app/controllers/organizations/adopter_fosterer/donations_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Organizations
+  module AdopterFosterer
+    class DonationsController < Organizations::BaseController
+      skip_verify_authorized only: [:index]
```

Comment:
no need to authorize accessing this route as it's re-showing a route with public information, just inside the dashboard layout